requirements.txt

```diff
@@ -1,4 +1,5 @@
 sphinx
 sphinx_rtd_theme
 sphinx-copybutton
-sphinx-design
\ No newline at end of file
+sphinx-design
+sphinxcontrib-video
```

Comment:
When using this package `sphinxcontrib-video` I got some warnings of deprecated modules like `sphinx.util.SkipProgressMessage` and `sphinx.util.progress_message`. I don't know if other people got the same warnings but I thought it is worth mentioning here in case something needs to be changed in the future.

Reply:
I just created a new environment and a clean build, and I no longer get this warning.